src/Controller/Admin/DashboardController.php

```diff
@@ -14,23 +14,47 @@
 use EasyCorp\Bundle\EasyAdminBundle\Config\Action;
 use EasyCorp\Bundle\EasyAdminBundle\Config\Actions;
 use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Config\UserMenu;
+use Symfony\Component\Security\Core\User\UserInterface;
+
 
 class DashboardController extends AbstractDashboardController
 {
     #[Route('/admin', name: 'admin')]
     public function index(): Response
     {
         $routeBuilder = $this->container->get(AdminUrlGenerator::class);
-        $url = $routeBuilder->setController(UserCrudController::class)->generateUrl();
+        $url = $routeBuilder->setController(DescriptionCrudController::class)->generateUrl();
 
         return $this->redirect($url);
     }
 
+    // #[Route('/admin')]
+    // public function index(): Response
+    // {
+    //     return $this->render('admin/my-dashboard.html.twig', []);
+    // }
```

Comment:
Remove unused code. Version control comes to the rescue if code has to be restored.